api/lib/domain/services/organization-invitation-service.js

```diff
@@ -33,6 +32,36 @@ const createOrganizationInvitation = async ({
   return organizationInvitation;
 };
 
+const createOrganizationSCOInvitation = async ({
```

Comment:
qu'en pensez-vous ?
```suggestion
const createSCOOrganizationInvitation = async ({
```

Reply:
J'ai homogénéisé partout en mettant scoOrganisation, comme on arrive pas à trancher :) je vous suggère cette proposition.